strider/trapi.py

```diff
@@ -89,17 +89,21 @@ def merge_edges(kedges: list[Edge]) -> Edge:
 def merge_kgraphs(kgraphs: list[KnowledgeGraph]) -> KnowledgeGraph:
     """ Merge knowledge graphs. """
 
+    if len(kgraphs) == 1:
+        return {
```

Comment:
Is there a reason we don't just return `kgraphs[0]`?

Reply:
1266bb9c0953890e0c31f0c4843649faab7bf513